docs/src/projects/configuration.md

```diff
@@ -16,6 +16,21 @@ exit_first = true
 
 Forge automatically looks for `Scarb.toml` in the directory you are running the tests in or in any of its parents.
 
+### Configuring Fork in `Scarb.toml`
+
+Forge makes things easier by letting you set up fork configurations inside the Scarb.toml file. Instead of repeatedly typing out the same details on the command line, you can define them in the file. 
+
+For example, you can set up different forks like this:
+
+```toml
+[[tool.snforge.fork]]
+name = "Starknet"
+url = "http://your.rpc.url"
+block_id.tag = "Latest"
+```
+
+To learn more, check out the page about [Fork Testing](../testing/fork-testing.md).
+
```

Comment:
I'm not sure if it makes sense to duplicate this information here. That's one more place to maintain it and I don't think anyone looking for information on how to configure forking would search in this file.

There's already info about that here https://github.com/foundry-rs/starknet-foundry/blob/0ea77905ea437a70b2861cd3476a7e8654c23927/docs/src/testing/fork-testing.md?plain=1#L45


```suggestion
```